src/pages/ExchangerDetails.jsx

```diff
@@ -0,0 +1,92 @@
+import determineColor from 'utils/determineColor';
+
+const ExchangerDetails = () => {
+  const location = useLocation();
+  const exchangeData = location.state;
+
+  if (!exchangeData) {
+    return <p>No exchange data provided.</p>;
+  }
+
+  return (
+    <Container className="mt-4 mb-4">
+      <Row className="justify-content-md-center">
+        <Col xs={12} md={10}>
+          <Card>
+            {/* Logo and name in the card header */}
+            <Card.Header className="d-flex justify-content-center align-items-center">
+              <Image src={exchangeData.image} alt={`${exchangeData.name} Logo`} width={50} height={50} roundedCircle className="me-3" />
+              <Card.Title className="mb-0">{exchangeData.name}</Card.Title>
+            </Card.Header>
+
+            {/* Other details in the card body */}
+            <Card.Body>
+              <Card.Text>
+                <strong>Country:</strong>
+                {' '}
+                {exchangeData.country}
+              </Card.Text>
+              <Card.Text>
+                <strong>Description:</strong>
+                {' '}
+                {exchangeData.description ? exchangeData.description : 'N/A'}
+              </Card.Text>
+              <Card.Text>
+                <strong>Trading Incentive:</strong>
+                {' '}
+                {exchangeData.has_trading_incentive ? 'Available' : 'Not Available'}
+              </Card.Text>
+              <Card.Text>
+                <strong>24h Trade Volume (BTC):</strong>
+                {' '}
+                {exchangeData.trade_volume_24h_btc.toFixed(2)}
+                {' '}
+                BTC
+              </Card.Text>
+              <Card.Text>
+                <strong>Normalized 24h Trade Volume (BTC):</strong>
+                {' '}
+                {exchangeData.trade_volume_24h_btc_normalized.toFixed(2)}
+                {' '}
+                BTC
+              </Card.Text>
+              <Card.Text>
+                <strong>Trust Score:</strong>
+                {' '}
+                <ProgressBar
+                  now={exchangeData.trust_score * 10}
+                  label={`${exchangeData.trust_score}/10`}
+                  variant={determineColor(exchangeData.trust_score)}
+                  style={{ flexGrow: 1 }}
+                />
+              </Card.Text>
+              <Card.Text>
+                <strong>Trust Score Rank:</strong>
+                {' '}
+                #
+                {exchangeData.trust_score_rank}
+              </Card.Text>
+              <Card.Text>
+                <strong>Established:</strong>
+                {' '}
+                {exchangeData.yearEstablished}
+              </Card.Text>
+              <Button variant="primary" href={exchangeData.url} target="_blank" rel="noopener noreferrer">
+                Visit
+                {' '}
+                {exchangeData.name}
+              </Button>
+            </Card.Body>
+          </Card>
```

Comment:
- Kindly note also, that after you make the changes I pointed out on the card's background color, you'd have to update your details page too so that each paragraph would have its respective background color as the same as for each card, and align the text content correctly.

*I used this first design to show you, please, follow your main design color*
![image](https://github.com/fmanimashaun/cryptorank-app/assets/59397159/dfc21c8d-8afa-4346-923d-17654dc013f3)


*your details page*
![image](https://github.com/fmanimashaun/cryptorank-app/assets/59397159/c58494cd-34e9-4cc6-96fc-bcf1ff6b8f69)

Reply:
![image](https://github.com/fmanimashaun/cryptorank-app/assets/22054464/75f63ecb-068c-4754-b4f3-62367a174450)


As requested, the necessary adjustment as been implemented as recommended